platforms/pictique/client/src/routes/(protected)/settings/account/+page.svelte

```
@@ -3,6 +3,6 @@
 	import { onMount } from 'svelte';
 
 	onMount(() => {
-		goto('/settings/account/username');
+		goto('/settings/account/username', { replaceState: true });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)